src/pages/commits/ActiveCommits/ChartToggle.tsx

```diff
@@ -3,6 +3,7 @@ import styled from "@emotion/styled";
 import { uiColors } from "@leafygreen-ui/palette";
```

Comment:
Padding looks a little off for the top-portion of the toggle (was this intentional)? 

<img width="1417" alt="Screen Shot 2022-02-08 at 4 14 12 PM" src="https://user-images.githubusercontent.com/47064971/153076760-210ae42d-907f-4388-8101-12ea854772b9.png">

Reply:
Great catch!